src/app/tests/suites/certification/Test_TC_DRLK_2_7.yaml

```diff
@@ -367,23 +343,24 @@ tests:
       response:
           error: INVALID_COMMAND
 
-    - label: "Clear a year day schedule for the first user"
-      PICS: DRLK.S.F0a && DRLK.S.C10.Rsp
-      command: "ClearYearDaySchedule"
+    - label: "Step 14:TH sends ClearUser Command to DUT with the UserIndex as 1"
+      PICS: DRLK.S.F08 && DRLK.S.C1d.Rsp
+      timedInteractionTimeoutMs: 10000
+      command: "ClearUser"
       arguments:
           values:
-              - name: "YearDayIndex"
-                value: 1
               - name: "UserIndex"
                 value: 1
 
-    - label: "Cleanup the created user with UserIndex 1"
-      command: "ClearUser"
-      timedInteractionTimeoutMs: 10000
+    - label: "Clear a year day schedule for the first user"
+      PICS: DRLK.S.F0a && DRLK.S.C10.Rsp
+      command: "ClearYearDaySchedule"
       arguments:
           values:
-              - name: "UserIndex"
+              - name: "YearDayIndex"
                 value: 1
+              - name: "UserIndex"
+                value: 5
 
     - label: "Cleanup the created user with UserIndex 5"
```

Comment:
Again, this is not in the test plan.  Why is it here?

Reply:
@bzbarsky-apple This step is added to cleanup the userindex 5 created in the step 7 to avoid the error. Now the test-plan is updated to clear all the users PR: https://github.com/CHIP-Specifications/chip-test-plans/pull/4571/files#diff-450e09b9acf3e7226af243b14d01c5bcc6f7a3acb5e39bda2726f22ba8dfcbdcR2165. So the step 14 is updated and removed this clean up step.